src/main.css

```diff
@@ -424,7 +424,7 @@ being. */
     bottom: 0;
     transform: translateX(-50%);
     width: 40%;
-    min-width: 560px;
+    min-width: 1100px;
```

Comment:
I'll have a think about this. It'd be nice if there was a way to do this without taking up so much horizontal space.

Reply:
I thought it might be nice to replace the text on each of these buttons with icons to save space, but it seemed like an unnecessary change for this feature.

Reply:
For now, we could save space by reducing the button width (and maybe making some of the text shorter), and moving "Export" outside the buttons, e.g.
![image](https://user-images.githubusercontent.com/3943692/106764234-82ddc480-662f-11eb-8232-a0947f939459.png)
Maybe "Link" suffices here.

Reply:
Yeah this definitely works!